src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2698,6 +2698,44 @@ def update_long_term_retention(
     return policy
 
 
+def update_short_term_retention(
+        client,
+        database_name,
+        server_name,
+        resource_group_name,
+        retention_days,
+        diffbackup_hours):
+    '''
+    Updates short term retention for live database
+    '''
+
+    policy = client.create_or_update(
+        database_name=database_name,
+        server_name=server_name,
+        resource_group_name=resource_group_name,
+        retention_days=retention_days,
+        diff_backup_interval_in_hours=diffbackup_hours)
+
+    return policy
```

Comment:
as you add support_no_wait flag for the command, you need to add sdk_no_wait() in custom function
```
# inside load_command_table(...)
with self.command_group('mymod', mymod_sdk) as g:
    g.custom_command('command2', 'do_something_2', supports_no_wait=True)

# inside custom.py
from azure.cli.core.util import sdk_no_wait
def do_something_2(cmd, arg1, arg2, no_wait=False):
    return sdk_no_wait(no_wait, client.create_or_update, arg1, arg2)
```

Reply:
changed, pl check.